test/formatting.coffee

```diff
@@ -198,6 +198,27 @@ test "#1495, method call chaining", ->
   ).join ', '
   eq 'a, b, c', result
 
+test "chaining after inline ternary", ->
+  throws -> CoffeeScript.compile """
+    if 0 then 1 else g
+      a: 42
+    .h()
+  """
+
+  # Wrap calls containing spilling ternary
+  f = (x) -> h: x
+  id = (x) -> x
+  result = f if true then 42 else id
+      a: 2
+  .h
```

Comment:
Nope, there is no difference. I can add a separate test if you want, but I think as long as this works it should be ok (it's hard to make this pass but your case fail).